### ISSUE TITLE
How to use "transl" in the dataset/parsed_data/rich_test_vit.pth

### ISSUE THREAD

**Issue by zehongs** [yohanshin/WHAM#13]
Hi @yohanshin,
Can you give me some suggestions on how to use the `transl` in `rich_test_vit.pth` to recover the ground truth global motion?

I found the `transl`s are the same for index 6 and 37 (`test/Gym_012_lunge1/cam_05` and `test/Gym_012_lunge1/cam_04`). `labels["transl"][6][1:] != labels["transl"][37][1:]).sum()` gives 0. Therefore, I think it means that the transl is in the world coordinate. However, the results of the computed joints3d are different from the provided joints3d (`joints3d = labels["joints3D"][index][1:]`). Could you help me to understand the differences?

```python
smplx_out = smplx_models[gender](
    body_pose=pose[:, 3:-6].reshape(-1, 63),
    global_orient=pose[:, :3],
    betas=betas,
)
smpl_verts = torch.matmul(smplx2smpl, smplx_out.vertices)  # in camera view, but zero-centered

transl_ = (cam_poses[:, :, :3] @ transl[:, :, None]  +cam_poses[:, :, 3:]).squeeze(2) 
smpl_verts = smpl_verts + transl_[:, None, :]
smpl_joints_c = smpl_J_regressor @ smpl_verts  # (F, 24, 3)
(smpl_joints_c - joints3d).abs().max()  # 0.0971

```

**Comment by yohanshin** [comment]
Hi @zehongs , Great observation! I just checked my preprocessing code and I realized that I did not transform the translation to the camera coordinate system. That's why the translations from different cameras are identical here. Instead, I transformed the global orientation to the camera coordinate and this is sufficient to evaluate `MPJPE`, `PA-MPJPE`, and `PVE`, therefore, the performance on those metrics will stay the same. But I admit that this parsed data is a bit confusing and I will update it later. Thanks for your feedback!
